README.md

```diff
@@ -0,0 +1,184 @@
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- Coming Soon .....
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get the content of this project locally you need to run the following commands in your terminal:
+
+and follow these steps.
+
+### Prerequisites
+
+- IDE(code editor) like: **Vscode**, **Sublime**, etc. 
+- [Git](https://www.linode.com/docs/guides/how-to-install-git-on-linux-mac-and-windows/) 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+1. Download the **Zip** file or clone the repo with:
+```bash git clone https://github.com/Iqbal-Elham/space-travelers-hub.git ``` 
+2. To access cloned directory run: 
+```bash cd space-travelers-hub ``` 
+3. Open it with the live server 
+
+### Install
+
+> To install linters and other project's dependencies run:
+```bash npm install ```
+
+### Usage
+
+To run the project, execute the following command:
+
+
+### Run tests
+
+To run tests, run the following command:
+
+Test for Linters For tracking linter errors locally you need to follow these steps: 
+1. For tracking linter errors in HTML files run:
+```bash npx hint . ``` 
+2. For tracking linter errors in CSS or SASS files run:
+```bash npx stylelint "**/*.{css,scss}" ``` 
+3. For tracking linter errors in javascript files run:
+```bash npx eslint . ``` 
+
+### Start the project
+
+Enter the follow code to run the project locally:
+
+`cd math-magician` then you can run `npm start` ; the project will run on port `3000`.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Iqbal Elham**
+
+- GitHub: [@iqbal-elham](https://github.com/Iqbal-Elham)
+- linkedIn: [@iqbal-elham](https://www.linkedin.com/in/iqbal-elham)
+
+👤 **Fithamlak Fikrie Lakew**
+
+- GitHub: [fithamlak](https://github.com/fithamlak)
+- linkedIn: [fithamlak-fikrie](https://www.linkedin.com/in/fithamlak-fikrie-942169225/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+- [ ] **Making Improvment**
+- [ ] **Deployment**
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+A great thanks to Microverse.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] The license file link is broken. Kindly add a license file and connect it to make the repository more professional.